Website/Packages/datax-pipeline/src/modules/flowDefinition/components/input/inputSettingsContent.jsx

```diff
@@ -415,7 +415,7 @@ export default class InputSettingsContent extends React.Component {
                     <Label className="ms-font-m" style={inlineBlockStyle}>
                         Describe Schema in JSON Format
                     </Label>
-                    <a style={linkStyle} href={inputSchemaExampleWiki} target="_blank">
+                    <a style={linkStyle} href={inputSchemaExampleWiki} target="_blank" rel="noopener noreferrer">
```

Comment:
can you check if the `Link` component from office ui fabric has this props by default or not? if yes, I'd suggest we move to use that component. 
If no, first let's search to see if we have used that in our project and also consider we make a similar common `Link` component to contain the shared code.

Reply:
office ui fabric support rel but they dont add rel="noopener noreferrer" by default when target is blank